projects/spectator/src/lib/select-option.ts

```diff
@@ -0,0 +1,49 @@
+import { dispatchFakeEvent } from './dispatch-events';
+import { isString } from './types';
+
+/**
+ * Focuses a select element, selects the correct options and dispatches
+ * the `change` event, simulating the user selecting an option
+ * @param options Options to be selected.
+ * @param element Element onto which to select the options.
+ *
+ * selectOption('al' | ['al', 'ab'], select);
+ */
+export function selectOption(options: string | string[], element: HTMLElement | HTMLSelectElement | Document | Window): void {
+  if (!(element instanceof HTMLSelectElement)) {
+    return;
+  }
+
+  element.focus();
+
+  if (isString(options)) {
+    const option: HTMLOptionElement | null = element.querySelector(`option[value="${options}"]`);
+    if (!option) {
+      return;
+    }
+
+    setOptionSelected(option, element);
+  } else {
+    if (!element.multiple) {
+      return;
+    }
+
+    element.querySelectorAll('option').forEach(opt => {
+      if (options.includes(opt.value)) {
+        setOptionSelected(opt, element);
```

Comment:
Do we need to dispatch the `change` event for each selected option?

Reply:
I figured that a developer might have an `onChange` handler set up that they are trying to test when certain options are selected.

Reply:
What's the behavior when using the native JS API?

Reply:
There is no native way of selecting multiple values of a select at once. From looking around online, a lot of solutions to this involve setting the selected option to true. 

It doesn't fire the change event simply by doing this. However, by firing the change event, I feel it's closer to simulating how a user would select the multiple values.

Perhaps this should be an option for the developer that they can turn off when using the method.

Reply:
> Perhaps this should be an option for the developer that they can turn off when using the method.

Agree.

Reply:
Added optional parameter and two more tests to cover it's usage.